mod/node-api/handlers/beacon/handler.go

```diff
@@ -41,8 +41,8 @@ type Handler[
 func NewHandler[
 	BeaconBlockHeaderT types.BeaconBlockHeader,
 	ContextT context.Context,
-	ForkT any,
-	ValidatorT any,
+	ForkT types.Fork,
```

Comment:
what is the purpose of these constraints?

Reply:
So the Backend can return responses of these interface types, which will help in using the getters directly in marshalling JSON. The idea is currently the response being returned for Validator and Fork is not matching with the Beacon API specification in terms of data types used, certain data types like uint64 needs to be modified to string and others, so that's why introduced these, so that we return the response as per Beacon API spec.